packages/store/src/internal/internals.ts

```diff
@@ -380,7 +380,7 @@ export function removeDollarAtTheEnd(name: string): string {
   return dollarAtTheEnd ? name.slice(0, lastCharIndex) : name;
 }
 
-export type MethodAccessorType = (
+export type MethodDecorator = (
```

Comment:
@splincode `MethodDecorator` is a built-in type

Reply:
Oh

Reply:
done